src/main/java/io/cdap/plugin/gcp/bigquery/sink/BigQuerySink.java

```diff
@@ -257,6 +266,21 @@ private long getTotalRows(Job queryJob) {
     return 0;
   }
 
+  private long getTotalBytes(Job queryJob) {
+    JobConfiguration.Type type = queryJob.getConfiguration().getType();
+    if (type == JobConfiguration.Type.LOAD) {
+      long outputBytes = ((JobStatistics.LoadStatistics) queryJob.getStatistics()).getOutputBytes();
+      LOG.info("Job {} loaded {} bytes", queryJob.getJobId(), outputBytes);
+      return outputBytes;
+    } else if (type == JobConfiguration.Type.QUERY) {
+      long processedBytes = ((JobStatistics.QueryStatistics) queryJob.getStatistics()).getTotalBytesProcessed();
+      LOG.info("Job {} processed {} bytes", queryJob.getJobId(), processedBytes);
+      return processedBytes;
+    }
+    LOG.warn("Unable to identify BigQuery job type. No metric will be emitted for the number of affected bytes.");
```

Comment:
is this ever expected? Would be good to add a comment in the code

Reply:
done